src/components/cylc/tree/nodes.js

```diff
@@ -332,7 +332,8 @@ function createCustomOutputs (job) {
  * @return {JobNode}
  */
 function createJobNode (job) {
-  Vue.set(job, 'customOutputs', createCustomOutputs(job))
+  const customOutputs = createCustomOutputs(job)
+  Vue.set(job, 'customOutputs', customOutputs)
```

Comment:
Note to self: I had this change in my IDE when debugging, to see the value of `customOutputs` and confirm it was valid. This can probably be undone now.